src/OSPSuite.Core/Converter/IObjectConverter.cs

```diff
@@ -6,20 +6,20 @@ namespace OSPSuite.Core.Converter
 {
    public interface IObjectConverter : ISpecification<int>
    {
-      int Convert(object objectToUpdate);
-      int ConvertXml(XElement element);
+      (int convertedToVersion, bool conversionHappened) Convert(object objectToUpdate);
```

Comment:
Instead of only version, we return a flag indicating whether a conversion happened or not. This will be used in other converter in PKSim and MoBi to set flags accordingly